AirCasting/SearchAndFollow/SearchMap/SearchMapView.swift

```diff
@@ -52,38 +68,33 @@ private extension SearchMapView {
     }
     
     var searchAgainButton: some View {
-            Button {
-                withAnimation(.easeOut(duration: 0.2)) {
-                    viewModel.redoTapped()
-                }
-            } label: {
-                Text("\(Strings.SearchMapView.redoText) \(Image(systemName: "goforward"))")
-                    .font(Fonts.boldHeading3)
-                    .lineLimit(1)
-                    .scaledToFill()
-                    .multilineTextAlignment(.leading)
-                    .padding(.horizontal, 10)
-                    .padding(.vertical, 6)
+        Button {
+            withAnimation(.easeOut(duration: 0.2)) {
+                viewModel.redoTapped()
             }
-            .foregroundColor(.white)
-            .background(Color.accentColor)
-            .cornerRadius(5)
-            .padding(-3)
-            .shadow(color: Color.shadow, radius: 9, x: 0, y: 1)
-            .disabled(!viewModel.searchAgainButton)
-            .opacity(viewModel.searchAgainButton ? 1.0 : 0.0)
+        } label: {
+            Text("\(Strings.SearchMapView.redoText) \(Image(systemName: "goforward"))")
+                .font(Fonts.boldHeading3)
+                .lineLimit(1)
+                .scaledToFill()
+        }
     }
     
     var map: some View {
-        GeometryReader { geometry in
+        GeometryReader { reader in
             ZStack(alignment: .top) {
                 SearchAndFollowMap(startingPoint: viewModel.passedLocationAddress,
-                                   showRedoButton: $viewModel.searchAgainButton,
-                                   sessions: $viewModel.sessionsList).onPositionChange(action: { geoSquare in
+                                   showSearchAgainButton: $viewModel.searchAgainButton,
+                                   sessions: $viewModel.sessionsList,
+                                   selectedPointerID: $viewModel.cardPointerID)
```

Comment:
It's not causing any issues, but I think that it is not necessary to pass the selectedPointerID to the map. I think it's a map that needs to inform the viewModel which marker was tapped, and it can do it with `onMarkerChange` function. Then the map itself can decide which marker should be larger, and it could be triggered in the `didTap` function (as I have sent you on slack). As we want to move forward with the feature, I think you could merge this PR anyway, but we could change this in another PR, to make code cleaner and remove unnecessary complexity.